plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -128,7 +128,8 @@ export class EventsProcessor {
             const personUuid = new UUIDT().toString()
 
             // TODO: we should just handle all person's related changes together not here and in capture separately
-            const ts = this.handleTimestamp(data, now, sentAt)
+            const parsedTs = this.handleTimestamp(data, now, sentAt)
+            const ts = parsedTs.isValid ? parsedTs : DateTime.now()
```

Comment:
Let's add another property or something that helps us later determine that we did this, would be helpful in debugging unexpected timestamps. Or maybe just statsd it by team